docs/config.md

```diff
@@ -66,6 +66,11 @@ Supported keys include:
 | Key | Description | Supported Values | Default |
 | :--- | :--- | :--- | :--- |
 | `signers.kms.kmsref` | The URI reference to a KMS service to use in `KMS` signers. | `gcpkms://projects/[PROJECT]/locations/[LOCATION]>/keyRings/[KEYRING]/cryptoKeys/[KEY]`| |
+| `signers.kms.auth.address` | URI of KMS server (e.g. the value of `VAULT_TOKEN`) | |
```

Comment:
could we reuse `signers.kms.kmsref`here?

Reply:
The KMS Ref ends up being something like `hashivault://keyname` while the address for the auth is the URL of the server itself, `http://vault:8200`. I am curious how this maps to other KMS solutions, so naming suggestions are welcome.